collector/config.yaml

```diff
@@ -7,17 +7,17 @@ receivers:
         endpoint: "localhost:4318"
 
 exporters:
-  logging:
```

Comment:
Do we need to remove an underlying dependency here to avoid a build failure in the future?

Reply:
In the future yes, I was going to leave that for now (as people transition away from it in their config)

Reply:
Ok.  I wasn't sure how quickly it was going to be removed.